query_sheet.rb

```diff
@@ -1,15 +1,41 @@
 #!/usr/bin/env ruby
 require 'json'
+require 'set'
```

Comment:
Could you update the README.md file to include this as a necessary import? Fine to do in a follow-up pull request.